packages/salesforcedx-vscode-core/test/vscode-integration/conflict/timestampConflictDetector.test.ts

```diff
@@ -24,6 +24,8 @@ import { nls } from '../../../src/messages';
 import { stubRootWorkspace } from '../util/rootWorkspace.test-util';
 
 describe('Timestamp Conflict Detector Execution', () => {
+  const TODAY = '2023-01-28T00:15:28.000Z';
+  const YESTERDAY = '2023-01-27T00:15:28.000Z';
```

Comment:
Would it be worth expanding this to test some other cases beside just yesterday and today?

Reply:
actually on 2nd thought let's def not expand the int tests.  If you move to the date comparaison I'd feel less worried about this validation.